src/instruction.rs

```diff
@@ -509,81 +509,49 @@ impl Instruction {
                 Register::M => 7,
                 _ => 4,
             },
-            Instruction::RNZ => 11, // FIX ME  11/5
+            Instruction::RNZ => 5,
```

Comment:
Choosing the lower number here as the higher number will be calculated when the condition's action is taken (see conditional_subroutine! within Instruction::emulate